cli/internal/publish/pluginui.go

```diff
@@ -107,22 +106,3 @@ func readFlatDir(base string) (files []string, err error) {
 	})
 	return files, err
 }
-
-func quickContentType(filename string) (string, error) {
```

Comment:
Moved this and the tests under `images` package. Seems more suitable now that it's re-used